compiler/rustc_hir/src/target.rs

```diff
@@ -185,4 +141,48 @@ impl Target {
             hir::GenericParamKind::Const { .. } => Target::GenericParam(GenericParamKind::Const),
         }
     }
+
+    pub fn name(&self) -> &str {
```

Comment:
This can take `self` by value, `Target` is a small enum which is `Copy`.
```suggestion
    pub fn name(self) -> &'static str {
```

Reply:
Resolved by 1e8abe7